client-data/js/board.js

```diff
@@ -59,6 +59,22 @@ if (window.location.pathname.includes("/robotboards/")) {
 }
 (function setupRobotTools() {
 	Tools.robotTools = RobotTools;
+	btn = document.getElementById("collabWhiteboard");
+	if (btn) {
+		btn.addEventListener("click", onTogglePageClick);
+	}
+	btn = document.getElementById("collabStation");
+	if (btn) {
+		btn.addEventListener("click", onTogglePageClick);
+	}
+	btn = document.getElementById("backButton");
```

Comment:
What is this? I can't find any such id.

Reply:
Thank you Mark, I changed the name at to match other naming type and added new event listener instead of updating older one. I will remove it